.github/hooks/pre-commit

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Not sure if we need pre commit hook, since we have github action for all checks already

Reply:
That is to be installed locally to pass the test before committing